config/tests/fixtures/full.env

```diff
@@ -0,0 +1,16 @@
+CHAIN_ID=0-0
+PUBLIC_KEY=ed01208BA62848CF767D72E7F7F4B9D2D7BA07FEE33760F79ABE5597A51520E292A0CB
+PRIVATE_KEY_DIGEST=ed25519
+PRIVATE_KEY_PAYLOAD=8f4c15e5d664da3f13778801d23d4e89b76e94c1b94b389544168b6cb894f84f8ba62848cf767d72e7f7f4b9d2d7ba07fee33760f79abe5597a51520e292a0cb
+P2P_ADDRESS=127.0.0.1:5432
+GENESIS_PUBLIC_KEY=ed01208BA62848CF767D72E7F7F4B9D2D7BA07FEE33760F79ABE5597A51520E292A0CB
+GENESIS_PRIVATE_KEY_DIGEST=ed25519
+GENESIS_PRIVATE_KEY_PAYLOAD=8f4c15e5d664da3f13778801d23d4e89b76e94c1b94b389544168b6cb894f84f8ba62848cf767d72e7f7f4b9d2d7ba07fee33760f79abe5597a51520e292a0cb
+API_ADDRESS=127.0.0.1:8080
+KURA_INIT_MODE=strict
+KURA_BLOCK_STORE=/store/path/from/env
```

Comment:
this seems a bit inconsistent. We have `GENESIS_FILE` (which I suggested to rename to `GENESIS_PATH`) but here we have no suffix. Either we should remove suffix from `GENESIS` or add a suffix `PATH`/`FILE` here

Reply:
What about using `FILE`/`DIR` suffixes?
- `GENESIS_FILE`
- `KURA_BLOCK_STORE_DIR`
- `SNAPSHOT_STORE_DIR`

Reply:
`DIR` is good. Why `FILE` and not `PATH`?

Reply:
> why "file" and not "path"

Because both `FILE` and `DIR` are of "path" category.

Reply:
After discussion, renamed to using `_file` and `_dir` prefixes.

`executor` in Genesis is remaining the same _for now_.